src/routes/(admin)/tests/test-[type]/[id]/QuestionDialog.svelte

```
@@ -0,0 +1,82 @@
+<script lang="ts">
+	import Info from '@lucide/svelte/icons/info';
+	import DataTable from './question_table/data-table.svelte';
+	import * as Dialog from '$lib/components/ui/dialog/index.js';
+	import Input from '$lib/components/ui/input/input.svelte';
+	import TagsSelection from '$lib/components/TagsSelection.svelte';
+	import StateSelection from '$lib/components/StateSelection.svelte';
+	let { open = $bindable(), questions, columns, formData } = $props();
+	let tags = $state<String[]>([]);
```

ESLint

[error] Prefer using the primitive `string` as a type name, rather than the upper-cased `String`.

(@typescript-eslint/no-wrapper-object-types)